scripts/velocity_test.py

```diff
@@ -34,42 +34,34 @@ def velocity_test():
 
     rospy.sleep(2.0)
 
-    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=0.2, y_velocity=-0.2)
+    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=0.2, y_velocity=0.0)
     # Sends the goal to the action server.
     client.send_goal(goal)
-    rospy.sleep(3.0)
-    client.cancel_goal()
-    rospy.logwarn("Translation into arena canceled")
     rospy.sleep(2.0)
-
-    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=0.3, y_velocity=0.0)
-    # Sends the goal to the action server.
-    client.send_goal(goal)
-    rospy.sleep(5.0)
     client.cancel_goal()
     rospy.logwarn("Translation 1 canceled")
     rospy.sleep(2.0)
 
-    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=-0.0, y_velocity=-0.3)
+    goal = QuadMoveGoal(movement_type="velocity_test", x_velocity=-0.0, y_velocity=-0.2)
```

Comment:
Since my comment didn’t make it to slack, can we get rid of the negative zeroes 😬

Reply:
Done